.github/workflows/prepare-release-branch.yml

```
@@ -0,0 +1,96 @@
+name: Prepare release branch
+on:
+  workflow_dispatch:
+
+jobs:
+  prepare-release-branch:
+    runs-on: ubuntu-latest
+    outputs:
+      release-branch-name: ${{ steps.set-release-branch-name.outputs.release-branch-name }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set release branch name
+        id: set-release-branch-name
+        run: |
+          version=$(grep -Eo "[0-9.]+-SNAPSHOT" version.gradle.kts)
+          release_branch_name=$(echo $version | sed -E 's/([0-9]+)\.([0-9]+)\.0/v\1.\2.x/')
+          echo "::set-output name=release-branch-name::$release_branch_name"
+
+      - name: Create release branch
+        env:
+          RELEASE_BRANCH_NAME: ${{ steps.set-release-branch-name.outputs.release-branch-name }}
+        run: |
+          git checkout -b $RELEASE_BRANCH_NAME
+          git push origin $RELEASE_BRANCH_NAME
+
+  create-pull-request-against-release-branch:
+    needs: prepare-release-branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.prepare-release-branch.outputs.release-branch-name }}
+
+      - name: Bump version on release branch
+        run: |
+          v=$(grep -Eo "[0-9]+.[0-9]+.0-SNAPSHOT" version.gradle.kts | sed 's/-SNAPSHOT//')
+          sed -ri "s/$PRIOR_VERSION-SNAPSHOT/$VERSION/" version.gradle.kts
+          sed -ri "s/$PRIOR_VERSION-alpha-SNAPSHOT/$VERSION-alpha/" version.gradle.kts
```

Neither `PRIOR_VERSION` nor `VERSION` are defined here; should it be
```suggestion
          version=$(grep -Eo "[0-9]+.[0-9]+.0-SNAPSHOT" version.gradle.kts | sed 's/-SNAPSHOT//')
          sed -ri "s/$version-SNAPSHOT/$version/" version.gradle.kts
          sed -ri "s/$version-alpha-SNAPSHOT/$version-alpha/" version.gradle.kts
```
instead?